---
title: "Publications"
permalink: /publications/
author_profile: true
encoding: UTF-8
---

{% include base_path %}

<style>
ul {
  list-style-type: none;
}
</style>

<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

## 2022

### 24\. Building consensus around the assessment and interpretation of symbiodiniaceae diversity
Davies S. W., Gamache M. H., Howe-Kerr L., Kriefall N.G., Baker A.C., Banaszak A., Bay L., Bellantuono A.J., Chan C. X., Claar D.C., Coffroth M.A., Cunning R., del Campo J., Frommlet J. C., Fuess L. E., Goulet T. L.,<ca>Hoadley K. D., Hume B. C. C., Kemp D. W., Kitchen S. A., LaJeunesse T. C., Lin S., McIlroy S., McMinds R., Nitschke M. R., Oakley C. A., Peixoto R. S., Prada C., Putnam H. M., Quigley K., Reich H. G., Reimer J. D., Rosales S., Saad O. S., Santos S. R., Shoguchi E., **Stephens T. G.**, Strader M. E., Suggett D. J., Swain T. D., Tran C., Traylor-Knowles N., Voolstra C. R., Weis V., Wright R., Yamashita H., Ziegler M., Correa A. M. S., and Parkinson J. E.. *Under review in PeerJ*, 2022. [[Preprint](https://doi.org/10.20944/preprints202206.0284.v1)]



### 23\. Endosymbiotic ratchet accelerates divergence after organelle origin
Bhattacharya D., Etten J. V., Benites L. F., and **Stephens T. G.**. *Under review in BioEssays*, 2022.



### 22\. Development of a portable test to diagnose coral stress
Meng Z., Williams A., Liau P., **Stephens T. G.**, Drury C., Chiles E. N., Su X., Javanmard M., and Bhattacharya D.. *Scientific Reports*, 12:14398, 2022. [[URL](https://doi.org/10.1038/s41598-022-18653-3)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41598-022-18653-3"></div>



### 21\. Multiple waves of viral invasions in Symbiodiniaceae algal genomes
Benites L. F., **Stephens T. G.**, and Bhattacharya D.. *Under review in Virus Evolution*, 2022. [[Preprint](https://doi.org/10.1101/2022.04.12.488082)]



### 20\. Loss of key endosymbiont genes may facilitate early host control of the chromatophore in *Paulinella*
\*Gabr A., \***Stephens T. G.**, and Bhattacharya D.. *iScience*, 25(9):104974, 2022.*Co-first authorship [[URL](https://doi.org/10.1016/j.isci.2022.104974)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.isci.2022.104974"></div>



### 19\. High-quality genome assembles from key Hawaiian coral species
**Stephens T. G.**, Lee J., Jeong Y., Yoon H. S., Putnam H. M., Majerova E., and Bhattacharya D.. *Under review in GigaScience*, 2022.



### 18\. Life on the edge: the Hawaiian model for coral evolution
Bhattacharya D., **Stephens T. G.**, Tinoco A., Richmond R., and Cleves P. A.. *Limnology and Oceanography*, 67:1976-1985, 2022. [[URL](https://doi.org/10.1002/lno.12181)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1002/lno.12181"></div>



### 17\. Retrotransposition facilitated the<ca>establishment<ca>of a primary plastid in<ca>the thecate<ca>amoeba<ca>*Paulinella*
\*Calatreva V., \***Stephens T. G.**, Gabr A., Grossman A. R., and Bhattacharya D.. *PNAS*, 119:23, e2121241119, 2022.*Co-first authorship [[URL](https://doi.org/10.1073/pnas.2121241119)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1073/pnas.2121241119"></div>



### 16\. Hypothesis: *Trans*-splicing generates evolutionary novelty in the photosynthetic amoeba *Paulinella*
\*Gabr A., \***Stephens T. G.**, and Bhattacharya D.. *Journal of Phycology*, 58:3, 392-405, 2022.*Co-first authorship [[URL](https://doi.org/10.1111/jpy.13247)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/jpy.13247"></div>



### 15\. Genome-powered classification: insights gained from coral algal symbionts
Dougan K. E., Gonzalez-Pech R. A., **Stephens T. G.**, Shah S., Chen Y., Ragan M. A., Bhattacharya D., and Chan C. X.. *Trends in Microbiology*, 30:9, 831-840, 2022. [[URL](https://doi.org/10.1016/j.tim.2022.02.001)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.tim.2022.02.001"></div>



### 14\. Evidence for a robust photosystem II in the photosynthetic amoeba *Paulinella*
Gabr A., Zournas A., **Stephens T. G.**, Dismukes G., and Bhattacharya D.. *New Phytologist*, 234:934-945, 2022. [[URL](https://doi.org/10.1111/nph.18052)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/nph.18052"></div>



## 2021

### 13\. Multi-omic characterization of the thermal stress phenome in the stony coral *Montipora capitata*
Williams A., Pathmanathan J. S., **Stephens T. G.**, Su X., Chiles E. N., Conetta D., Putnam H. M., and Bhattacharya D.. *PeerJ*, 9:e12335, 2021. [[Preprint](https://doi.org/10.1101/2021.02.05.429981)] [[URL](https://doi.org/10.7717/peerj.12335)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.7717/peerj.12335"></div>



### 12\. Why is primary endosymbiosis so rare?
**Stephens T. G.**, Calatreva V., Gabr A., Grossman A. R., and Bhattacharya D.. *New Phytologist*, 231:5, 1693-1699, 2021. [[URL](https://doi.org/10.1111/nph.17478)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/nph.17478"></div>



### 11\. Inferring phylogenomic relationship of microbes using scalable alignment-free methods
Bernard G., **Stephens T. G.**, Gonzalez-Pech R. A., and Chan C. X.. *In: Mengoni A., Bacci G., Fondi M. (eds) Bacterial Pangenomics. Methods in Molecular Biology Series, vol 2242*, 2021. [[URL](https://doi.org/10.1007/978-1-0716-1099-2_5)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1007/978-1-0716-1099-2_5"></div>



### 10\. Comparative genomics supports that Brazilian bioethanol *Saccharomyces cerevisiae* comprise a unified group of domesticated strains related to cacha<U+008D>a spirit yeasts
Jacobus A. P., **Stephens T. G.**, Youssef P., Gonzalez-Pech R., Ciccotosto-Camp M. M., Dougan K. E., Chen Y., Basso L. C., Frazzon J., Chan C. X. and Gross J.. *Frontiers in Microbiology*, 12:644089, 687, 2021. [[Preprint](https://doi.org/10.1101/2020.12.15.422965)] [[URL](https://doi.org/10.3389/fmicb.2021.644089)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.3389/fmicb.2021.644089"></div>



### 9\. Comparison of 15 dinoflagellate genomes reveals extensive sequence and structural divergence in family Symbiodiniaceae and genus *Symbiodinium*
Gonzalez-Pech, R. A., **Stephens T. G.**, Chen Y., Mohamed A. R., Cheng Y., Shah S., Dougan K. E., Fortuin M. D. A., Lagorce R., Burt D. W., Bhattacharya D., Ragan M. A. and Chan C. X.. *BMC Biology*, 19:1, 73, 2021. [[Preprint](https://doi.org/10.1101/783902)] [[URL](https://doi.org/10.1186/s12915-021-00994-6)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1186/s12915-021-00994-6"></div>



## 2020

### 8\. Amoeba genome reveals dominant host contribution to plastid endosymbiosis
Lhee D., Lee J., Ettahi K., Cho C. H., Ha J., Chan Y., Zelzion U., **Stephens T. G.**, Price D. C., Gabr A., Nowack E. C. M., Bhattacharya D. and Yoon H. S.. *Molecular Biology and Evolution*, 38:2, 344<d0>357, 2020. [[URL](https://doi.org/10.1093/molbev/msaa206)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/molbev/msaa206"></div>



### 7\. Genomes of the dinoflagellate *Polarella glacialis* encode tandemly repeated single-exon genes with adaptive functions
**Stephens T. G.**, Gonzalez-Pech R. A., Cheng Y., Mohamed A. R., Burt D. W., Bhattacharya D., Ragan M. A. and Chan C. X.. *BMC Biology*, 18:56, 2020. [[Preprint](https://doi.org/10.1101/704437)] [[URL](https://doi.org/10.1186/s12915-020-00782-8)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1186/s12915-020-00782-8"></div>



## 2019

### 6\. Evidence that inconsistent gene prediction can mislead analysis of dinoflagellate genomes
Chen Y., Gonzalez?Pech R. A., **Stephens T. G.**, Bhattacharya D. and Chan C. X.. *Journal of Phycology*, 56, 6-10, 2019. [[Preprint](https://doi.org/10.1101/690040)] [[URL](https://doi.org/10.1111/jpy.12947)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/jpy.12947"></div>



## 2018

### 5\. Commonly misunderstood parameters of NCBI BLAST and important considerations for users
\*Gonzalez-Pech R. A., \***Stephens T. G.**, and Chan C. X.. *Bioinformatics*, bty1018, 2018.*Co-first authorship [F1000 recommended] [[URL](https://doi.org/10.1093/bioinformatics/bty1018)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/bioinformatics/bty1018"></div>



### 4\. Core genes in diverse dinoflagellate lineages include a wealth of conserved dark genes with unknown functions
**Stephens T. G.**, Ragan M. A., Bhattacharya D. and Chan C. X.. *Scientific Reports*, 8:1, 17175, 2018. [[URL](https://doi.org/10.1038/s41598-018-35620-z)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41598-018-35620-z"></div>



### 3\. *Symbiodinium* genomes reveal adaptive evolution of functions related to coral-dinoflagellate symbiosis
Liu H., **Stephens T. G.**, Gonzalez-Pech R. A., Beltran V. H., Lapeyre B., Bongaerts P., Cooke I., Aranda M., Bourne D. G., Foret S., Miller D. J., van Oppen M. J. H., Voolstra C. R., Ragan M.A. and Chan C.X.. *Communications Biology*, 1:95, 2018. [[Preprint](https://doi.org/10.1101/198762)] [[URL](https://doi.org/10.1038/s42003-018-0098-3)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s42003-018-0098-3"></div>



### 2\. Analysis of the draft genome of the red seaweed *Gracilariopsis chorda* provides insights into genome size evolution in Rhodophyta
Lee J, Yang E. C., Graf L., Yang J. H., Qiu H., Zelzion U., Chan C. X., **Stephens T. G.**, Weber A. P. M., Boo G. H., Boo S. M., Kim K. M., Shin Y., Jung M., Lee S. J., Yim H. S., Lee J. Y., Bhattacharya D. and Yoon H. S.. *Molecular Biology and Evolution*, 35:1869-1886, 2018. [[URL](https://doi.org/10.1093/molbev/msy081)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/molbev/msy081"></div>



## 2016

### 1\. PhySortR: a fast, flexible tool for sorting phylogenetic trees in R
**Stephens T. G.**, Bhattacharya D., Ragan M. A. and Chan C. X.. *PeerJ*, 4:e2038, 2016. [[Preprint](https://doi.org/10.7287/peerj.preprints.1609v1)] [[URL](https://doi.org/10.7717/peerj.2038)]
<div class='altmetric-embed' data-badge-type='donut' data-doi="10.7717/peerj.2038"></div>



