---
title: "Publications"
permalink: /publications/
author_profile: true
encoding: UTF-8
---

{% include base_path %}

<style>
ul {
  list-style-type: none;
}
</style>

<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

## 2024

### 34\. Fpa (YlaN) is an iron(II) binding protein that functions to relieve Fur-mediated repression of gene expression in *Staphylococcus aureus*
Boyd J. M, Kaler K. R., Esquilin-Lebron K., Pall A., Campbell C. J., Foley M. E., Rios-Delgado G., Mustor E. M., **Stephens T. G.**, Bovermann H., Greco T. M., Cristea I. M., Carabetta V. J., Beavers W. N., Bhattacharya D., Skaar E. P., Shaw L. N., Stemmler T. L. *mBio*, 2024.

[[URL](https://doi.org/10.1128/mbio.02310-24)]  [[Preprint](https://doi.org/10.1101/2023.10.03.560778)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mbio.02310-24"></div>
<br/><br/><br/><br/>
### 33\. What has *Paulinella* taught us about endosymbiont metabolic integration?
Calatrava V.\*, **Stephens T. G.**\*, Grossman A. R., and Bhattacharya D. *Schwartzbach, S.D., Kroth, P.G., Obornik, M. (eds) Endosymbiotic Organelle Acquisition. Springer, Cham.*, 2024.

[[URL](https://doi.org/10.1007/978-3-031-57446-7_12)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1007/978-3-031-57446-7_12"></div>
<br/><br/><br/><br/>
### 32\. Gene expression response under thermal stress in two Hawaiian corals is dominated by ploidy and genotype
Chille E. E., **Stephens T. G.**, Misri D., Strand E. L, Putnam H. M., and Bhattacharya D. *Ecology and Evolution*, 14:e70037, 2024.

[[URL](https://doi.org/10.1002/ece3.70037)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1002/ece3.70037"></div>
<br/><br/><br/><br/>
### 31\. Evidence of a putative CO2 delivery system to the chromatophore in the photosynthetic amoeba *Paulinella*
Gabr A., **Stephens T. G.**, Reinfelder J. R., Liau P., Calatrava V., Grossman A. R., and Bhattacharya D. *Environmental Microbiology Reports*, e13304, 2024.

[[URL](https://doi.org/10.1111/1758-2229.13304)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/1758-2229.13304"></div>
<br/><br/><br/><br/>
### 30\. Diverse fates of ancient horizontal gene transfers in extremophilic red algae
Etten J. V., **Stephens T. G.**, Chille E., Lipzen A., Peterson D., Barry K., Grigoriev I. V., and Bhattacharya D. *Environmental Microbiology*, 26(5):e16629, 2024.

[[URL](https://doi.org/10.1111/1462-2920.16629)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/1462-2920.16629"></div>
<br/><br/><br/><br/>
### 29\. Viruses associated with extremophilic red algal mats reveal signatures of early thermal adaptation
Benites L. F., **Stephens T. G.**, Etten J. V., James T., Christian W. C., Barry K., Grigoriev I. V., McDermott T. R. & Bhattacharya D. *Communications Biology*, 2024.

[[URL](https://doi.org/10.1038/s42003-024-05931-1)]  
Featured by <a href="https://www.science.org/content/article/giant-viruses-played-key-role-early-life-study-yellowstone-hot-spring-suggests?utm_source=sfmc&utm_medium=email&utm_campaign=ScienceAdviser&utm_content=distillation&et_rid=960447322&et_cid=5171391" target="_blank">Science</a>

<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s42003-024-05931-1"></div>
<br/><br/><br/><br/>
## 2023

### 28\. Facultative lifestyle drives diversity of coral algal symbionts
Bhattacharya D., **Stephens T. G.**, Chille E. E., Benites L. F., Chan C. X. *Trends in Ecology and Evolution*, 39(3):239-247, 2023.

[[URL](https://doi.org/10.1016/j.tree.2023.10.005)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.tree.2023.10.005"></div>
<br/><br/><br/><br/>
### 27\. Peeling back the layers of coral holobiont multi-omics data
\*Williams A., \***Stephens T. G.**, Shumaker A., and Bhattacharya D. *iScience*, 26(9):107623, 2023.

[[URL](https://doi.org/10.1016/j.isci.2023.107623)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.isci.2023.107623"></div>
<br/><br/><br/><br/>
### 26\. Ploidy variation and its implications for reproduction and population dynamics in two sympatric Hawaiian coral species
**Stephens T. G.**, Strand E. L., Putnam H. M., and Bhattacharya D. *Genome Biology and Evolution*, evad149, 2023.

[[URL](https://doi.org/10.1093/gbe/evad149)]  [[Preprint](https://doi.org/10.1101/2021.11.21.469467)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/gbe/evad149"></div>
<br/><br/><br/><br/>
### 25\. A *k*-mer-based approach for inferring phylogenetic relatedness of environmental genomic data
Etten J. V, **Stephens T. G.**, and Bhattacharya D. *Systems Biology*, syad037, 2023.

[[URL](https://doi.org/10.1093/sysbio/syad037)]  
Won the <a href="https://www.systbio.org/publishers-award.html" target="_blank">Publisher's Award for Excellence in Systematic Research</a>

<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/sysbio/syad037"></div>
<br/><br/><br/><br/>
### 24\. Algae obscura: The potential of rare species as model systems
Etten J. V., Benites F. L., **Stephens T. G.**, Yoon H. S., and Bhattacharya D. *Journal of Phycology*, 59(2):293-300, 2023.

[[URL](https://doi.org/10.1111/jpy.13321)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/jpy.13321"></div>
<br/><br/><br/><br/>
## 2022

### 23\. High-quality genome assemblies from key Hawaiian coral species
**Stephens T. G.**, Lee J., Jeong Y., Yoon H. S., Putnam H. M., Majerova E., and Bhattacharya D. *GigaScience*, 11:giac098, 2022.

[[URL](https://doi.org/10.1093/gigascience/giac098)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/gigascience/giac098"></div>
<br/><br/><br/><br/>
### 22\. Multiple waves of viral invasions in Symbiodiniaceae algal genomes
Benites L. F., **Stephens T. G.**, and Bhattacharya D. *Virus Evolution*, 8:veac101, 2022.

[[URL](https://doi.org/10.1093/ve/veac101)]  [[Preprint](https://doi.org/10.1101/2022.04.12.488082)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/ve/veac101"></div>
<br/><br/><br/><br/>
### 21\. Endosymbiotic ratchet accelerates divergence after organelle origin
Bhattacharya D., Etten J. V., Benites L. F., and **Stephens T. G.** *BioEssays*, e2200165, 2022.

[[URL](https://doi.org/10.1002/bies.202200165)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1002/bies.202200165"></div>
<br/><br/><br/><br/>
### 20\. Loss of key endosymbiont genes may facilitate early host control of the chromatophore in *Paulinella*
\*Gabr A., \***Stephens T. G.**, and Bhattacharya D. *iScience*, 25:104974, 2022.

[[URL](https://doi.org/10.1016/j.isci.2022.104974)]  
*Co-first authorship

<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.isci.2022.104974"></div>
<br/><br/><br/><br/>
### 19\. Development of a portable toolkit to diagnose coral thermal stress
Meng Z., Williams A., Liau P., **Stephens T. G.**, Drury C., Chiles E. N., Su X., Javanmard M., and Bhattacharya D. *Scientific Reports*, 12:14398, 2022.

[[URL](https://doi.org/10.1038/s41598-022-18653-3)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41598-022-18653-3"></div>
<br/><br/><br/><br/>
### 18\. Life on the edge: Hawaiian model for coral evolution
Bhattacharya D., **Stephens T. G.**, Tinoco A., Richmond R., and Cleves P. A. *Limnology and Oceanography*, 67:1976-1985, 2022.

[[URL](https://doi.org/10.1002/lno.12181)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1002/lno.12181"></div>
<br/><br/><br/><br/>
### 17\. Retrotransposition facilitated the establishment of a primary plastid in the thecate amoeba *Paulinella*
\*Calatreva V., \***Stephens T. G.**, Gabr A., Grossman A. R., and Bhattacharya D. *PNAS*, 119:e2121241119, 2022.

[[URL](https://doi.org/10.1073/pnas.2121241119)]  
*Co-first authorship

<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1073/pnas.2121241119"></div>
<br/><br/><br/><br/>
### 16\. Hypothesis: *Trans*-splicing generates evolutionary novelty in the photosynthetic amoeba *Paulinella*
\*Gabr A., \***Stephens T. G.**, and Bhattacharya D. *Journal of Phycology*, 58:392-405, 2022.

[[URL](https://doi.org/10.1111/jpy.13247)]  
*Co-first authorship

<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/jpy.13247"></div>
<br/><br/><br/><br/>
### 15\. Genome-powered classification of microbial eukaryotes: focus on coral algal symbionts
Dougan K. E., Gonzalez-Pech R. A., **Stephens T. G.**, Shah S., Chen Y., Ragan M. A., Bhattacharya D., and Chan C. X. *Trends in Microbiology*, 30:831-840, 2022.

[[URL](https://doi.org/10.1016/j.tim.2022.02.001)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.tim.2022.02.001"></div>
<br/><br/><br/><br/>
### 14\. Evidence for a robust photosystem II in the photosynthetic amoeba *Paulinella*
Gabr A., Zournas A., **Stephens T. G.**, Dismukes G., and Bhattacharya D. *New Phytologist*, 234:934-945, 2022.

[[URL](https://doi.org/10.1111/nph.18052)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/nph.18052"></div>
<br/><br/><br/><br/>
## 2021

### 13\. Multi-omic characterization of the thermal stress phenome in the stony coral *Montipora capitata*
Williams A., Pathmanathan J. S., **Stephens T. G.**, Su X., Chiles E. N., Conetta D., Putnam H. M., and Bhattacharya D. *PeerJ*, 9:e12335, 2021.

[[URL](https://doi.org/10.7717/peerj.12335)]  [[Preprint](https://doi.org/10.1101/2021.02.05.429981)]  
Featured by PeerJ Expert Curations: Molecular Ecology: <a href="https://peerj.com/special-issues/140-molecular-ecology" target="_blank">PeerJ Expert Curations</a>, <a href="https://peerj.com/blog/post/115284888922/molecular-ecology" target="_blank">PeerJ Blog</a>

<div class='altmetric-embed' data-badge-type='donut' data-doi="10.7717/peerj.12335"></div>
<br/><br/><br/><br/>
### 12\. Why is primary endosymbiosis so rare?
**Stephens T. G.**,  Gabr A., Calatreva V., Grossman A. R., and Bhattacharya D. *New Phytologist*, 231:1693-1699, 2021.

[[URL](https://doi.org/10.1111/nph.17478)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/nph.17478"></div>
<br/><br/><br/><br/>
### 11\. Inferring phylogenomic relationship of microbes using scalable alignment-free methods
Bernard G., **Stephens T. G.**, Gonzalez-Pech R. A., and Chan C. X. *Methods in Molecular Biology*, 2242:69-76, 2021.

[[URL](https://doi.org/10.1007/978-1-0716-1099-2_5)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1007/978-1-0716-1099-2_5"></div>
<br/><br/><br/><br/>
### 10\. Comparative genomics supports that Brazilian bioethanol *Saccharomyces cerevisiae* comprise a unified group of domesticated strains related to cachaca spirit yeasts
Jacobus A. P., **Stephens T. G.**, Youssef P., Gonzalez-Pech R., Ciccotosto-Camp M. M., Dougan K. E., Chen Y., Basso L. C., Frazzon J., Chan C. X., and Gross J. *Frontiers in Microbiology*, 12:644089, 2021.

[[URL](https://doi.org/10.3389/fmicb.2021.644089)]  [[Preprint](https://doi.org/10.1101/2020.12.15.422965)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.3389/fmicb.2021.644089"></div>
<br/><br/><br/><br/>
### 9\. Comparison of 15 dinoflagellate genomes reveals extensive sequence and structural divergence in family Symbiodiniaceae and genus *Symbiodinium*
Gonzalez-Pech, R. A., **Stephens T. G.**, Chen Y., Mohamed A. R., Cheng Y., Shah S., Dougan K. E., Fortuin M. D. A., Lagorce R., Burt D. W., Bhattacharya D., Ragan M. A., and Chan C. X. *BMC Biology*, 0.842361111, 2021.

[[URL](https://doi.org/10.1186/s12915-021-00994-6)]  [[Preprint](https://doi.org/10.1101/783902)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1186/s12915-021-00994-6"></div>
<br/><br/><br/><br/>
## 2020

### 8\. Amoeba genome reveals dominant host contribution to plastid endosymbiosis
Lhee D., Lee J., Ettahi K., Cho C. H., Ha J., Chan Y., Zelzion U., **Stephens T. G.**, Price D. C., Gabr A., Nowack E. C. M., Bhattacharya D., and Yoon H. S. *Molecular Biology and Evolution*, 38:344-357, 2020.

[[URL](https://doi.org/10.1093/molbev/msaa206)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/molbev/msaa206"></div>
<br/><br/><br/><br/>
### 7\. Genomes of the dinoflagellate *Polarella glacialis* encode tandemly repeated single-exon genes with adaptive functions
**Stephens T. G.**, Gonzalez-Pech R. A., Cheng Y., Mohamed A. R., Burt D. W., Bhattacharya D., Ragan M. A., and Chan C. X. *BMC Biology*, 18:56, 2020.

[[URL](https://doi.org/10.1186/s12915-020-00782-8)]  [[Preprint](https://doi.org/10.1101/704437)]  
Featured in: <a href="https://imb.uq.edu.au/article/2020/07/algae-use-distinct-method-when-swapping-dna-during-sex" target="_blank">IMB News: Algae use distinct method when swapping DNA during sex</a>

<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1186/s12915-020-00782-8"></div>
<br/><br/><br/><br/>
## 2019

### 6\. Evidence that inconsistent gene prediction can mislead analysis of dinoflagellate genomes
Chen Y., Gonzalez-Pech R. A., **Stephens T. G.**, Bhattacharya D., and Chan C. X. *Journal of Phycology*, 56:6-10, 2019.

[[URL](https://doi.org/10.1111/jpy.12947)]  [[Preprint](https://doi.org/10.1101/690040)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1111/jpy.12947"></div>
<br/><br/><br/><br/>
## 2018

### 5\. Commonly misunderstood parameters of NCBI BLAST and important considerations for users
\*Gonzalez-Pech R. A., \***Stephens T. G.**, and Chan C. X. *Bioinformatics*, 35:2697-6998, 2018.

[[URL](https://doi.org/10.1093/bioinformatics/bty1018)]  
*Co-first authorship [F1000 recommended]

<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/bioinformatics/bty1018"></div>
<br/><br/><br/><br/>
### 4\. Core genes in diverse dinoflagellate lineages include a wealth of conserved dark genes with unknown functions
**Stephens T. G.**, Ragan M. A., Bhattacharya D., and Chan C. X. *Scientific Reports*, 8:17175, 2018.

[[URL](https://doi.org/10.1038/s41598-018-35620-z)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41598-018-35620-z"></div>
<br/><br/><br/><br/>
### 3\. *Symbiodinium* genomes reveal adaptive evolution of functions related to coral-dinoflagellate symbiosis
Liu H., **Stephens T. G.**, Gonzalez-Pech R. A., Beltran V. H., Lapeyre B., Bongaerts P., Cooke I., Aranda M., Bourne D. G., Foret S., Miller D. J., van Oppen M. J. H., Voolstra C. R., Ragan M.A., and Chan C.X. *Communications Biology*, 0.107638889, 2018.

[[URL](https://doi.org/10.1038/s42003-018-0098-3)]  [[Preprint](https://doi.org/10.1101/198762)]  
Featured in:<br /> - <a href="https://imb.uq.edu.au/article/2018/07/unlocking-genetic-secrets-help-save-great-barrier-reef" target="_blank">IMB News: Unlocking genetic secrets to help save the Great Barrier Reef</a><br /> - <a href="https://www.barrierreef.org/latest/news/decoding-the-reef" target="_blank">GBRF: Decoding the Reef leads to world first. Unlocking genetic secrets of coral symbiosis to preserve iconic Great Barrier Reef</a><br /> - <a href="https://youtu.be/k7Dg6D1hXtk" target="_blank">Video Feature</a><br /> - <a href="https://youtu.be/HimliKBZ-eY" target="_blank">Featured in IMB 2018 year in review</a>

<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s42003-018-0098-3"></div>
<br/><br/><br/><br/>
### 2\. Analysis of the draft genome of the red seaweed *Gracilariopsis chorda* provides insights into genome size evolution in Rhodophyta
Lee J, Yang E. C., Graf L., Yang J. H., Qiu H., Zelzion U., Chan C. X., **Stephens T. G.**, Weber A. P. M., Boo G. H., Boo S. M., Kim K. M., Shin Y., Jung M., Lee S. J., Yim H. S., Lee J. Y., Bhattacharya D., and Yoon H. S. *Molecular Biology and Evolution*, 35:1869-1886, 2018.

[[URL](https://doi.org/10.1093/molbev/msy081)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/molbev/msy081"></div>
<br/><br/><br/><br/>
## 2016

### 1\. PhySortR: a fast, flexible tool for sorting phylogenetic trees in R
**Stephens T. G.**, Bhattacharya D., Ragan M. A., and Chan C. X. *PeerJ*, 4:e2038, 2016.

[[URL](https://doi.org/10.7717/peerj.2038)]  [[Preprint](https://doi.org/10.7287/peerj.preprints.1609v1)]  


<div class='altmetric-embed' data-badge-type='donut' data-doi="10.7717/peerj.2038"></div>
<br/><br/><br/><br/>

